test/renate/utils/test_pytorch.py

```diff
@@ -11,6 +11,7 @@
 from renate.memory.buffer import ReservoirBuffer
 from renate.utils import pytorch
 from renate.utils.pytorch import (
```

Comment:
Is it possible to add a `DistributedSampler` to a test?

Reply:
I've added a unit test for the distributed case instead